omni/omni.py

```
@@ -0,0 +1,956 @@
+                try:
+                    await message.delete()
+                    self.memory_moderated_users[guild.id][message.author.id] += 1
+                    message_deleted = True
+                except discord.NotFound:
+                    pass
+
+            timeout_issued = False
+            if timeout_duration > 0:
+                try:
+                    reason = (
+                        f"The AI moderator issued a timeout. Violation scores: " +
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)